spec.bs

```diff
@@ -2800,6 +2803,143 @@ This specification defines two [=policy-controlled features=] identified by the
 Issue(WICG/turtledove#522): Move from "`*`" to "`self`".
 
 
+# Handling Direct from Seller Signals # {#handling-direct-from-seller-signals}
+
+This section specifies a manner by which signals may be provided to auctions such that the signals
+are only used within their intended auction.
+
+Scripts running on the page may peform a `fetch()` call using the `{adAuctionHeaders: true}`
+option, which causes the value of the `Ad-Auction-Signals` response header on that request to
+be used to provide signals in auctions running on the page whose seller matches the origin of the
+`fetch()` destination.
+
+<div algorithm="fetch adAuctionHeaders patch">
+Modify [[FETCH]]'s [[FETCH#requests]] to add: 
+
+A [=request=] has an associated <dfn for=request>capture ad auction headers boolean</dfn>.
+Unless stated otherwise it is false.
+
+Modify [[FETCH]]'s <a href="https://fetch.spec.whatwg.org/#requestinit">RequestInit</a> dictionary to add
+an adAuctionHeaders attribute:
+
+<pre class=idl>
+partial dictionary RequestInit {
+  boolean adAuctionHeaders;
+};
+</pre>
+
+The following step will be added to the <a href="https://fetch.spec.whatwg.org/#dom-request">new Request(input, init) constructor steps</a>,

```

Comment:
When in doubt, do not link to this with an anchor tag. https://github.com/whatwg/fetch/blob/main/fetch.bs#L7438-L7439 is how you can link to this constructor.

Reply:
I couldn't get that to work, for some reason, and this syntax I found in the trust token spec didn't work either [0].

My code reads:

```The following step will be added to the <code><a constructor lt="Request()">new Request (<var ignore>input</var>, <var ignore>init</var>)</a></code> constructor steps, before step "Set this's [=Request/request=] to |request|":```

I get this error: 

```
LINK ERROR: No 'constructor' refs found for 'Request()' with for='None'.
<a bs-line-number="2864" data-lt="Request()" data-link-type="constructor">new Request (<var bs-line-number="2864" data-var-ignore="">input</var>, <var bs-line-number="2864" data-var-ignore="">init</var>)</a>
 ✔  Successfully generated, with 1 linking errors
```

I tried copying some of the fetch related lines from the anchors and link-defaults from the trust token spec, but it either didn't help, or I got more errors ;) 

[0] https://github.com/WICG/trust-token-api/blob/main/spec.bs#L597.

Reply:
Oops, looks like you'll need `for=Request`

Reply:
That worked, thanks :)